dpppt-config-backend/src/main/java/org/dpppt/switzerland/backend/sdk/config/ws/config/WSBaseConfig.java

```diff
@@ -120,4 +128,104 @@ private PublicKey loadPublicKeyFromString() {
 			throw new RuntimeException();
 		}
 	}
+
+    @Bean
+    public Messages messages(MessageSource messageSource) {
+		Messages messages = new Messages(messageSource);
+		loadTexts(messages);
+		return messages;
+    }
+
+    @Bean
+    public MessageSource messageSource() {
+        ReloadableResourceBundleMessageSource messageSource =
+                new ReloadableResourceBundleMessageSource();
+
+        messageSource.setBasename("classpath:i18n/messages");
+        messageSource.setDefaultEncoding("UTF-8");
+        return messageSource;
+    }
+
+    public void loadTexts(Messages messages) {
+        ConfigResponse.setWhatToDoPositiveTestTexts(
+                new WhatToDoPositiveTestTextsCollection() {
+                    {
+						setDe(getWhatToDoPositiveTestText(messages, Locale.forLanguageTag("de")));
```

Comment:
I think it would be reasonable to have a UnitTest testing that forLanguageTag returns the correct language. 
It never fails but returns the key in the worst case, so we won't notice until we get feedback...

Reply:
@ubhaller @UBaggeler

Reply:
I decided to write a unit test checking a text for every language instead of testing forLanguageTag.